recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
         - python
         - pytz
         - billiard >=3.5.0.2,<3.6
-        - kombu >=4.0.2,<5.0
+        - kombu >=4.1.0,<5.0
```

Comment:
Yeah - this is the result of  find/replace in the script. It's why vetting is important!